modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/HectorBasedMessageStoreImpl.java

```diff
@@ -490,26 +454,80 @@ public AndesMessageMetadata getMetaData(long messageId) throws AndesException {
      * {@inheritDoc}
      */
     @Override
-    public void deleteMessageMetadataFromQueue(String queueName, List<AndesRemovableMetadata>
+    public void deleteMessageMetadataFromQueue(String queueName, List<Long>
             messagesToRemove) throws AndesException {
 
         Context context = MetricManager.timer(Level.DEBUG, MetricsConstants.DELETE_MESSAGE_META_DATA_FROM_QUEUE).start();
         try {
             if (log.isTraceEnabled()) {
                 StringBuilder messageIDsString = new StringBuilder();
-                for (AndesRemovableMetadata metadata : messagesToRemove) {
-                    messageIDsString.append(metadata.getMessageID()).append(" , ");
+                for (Long metadata : messagesToRemove) {
+                    messageIDsString.append(metadata.longValue()).append(" , ");
```

Comment:
It is only used in one other place (used for the same purpose of appending messageIDs), should I still use a constant?